integration_tests/utils.py

```
@@ -1029,22 +1031,50 @@ async def assert_tf_flow(w3, receiver, signer1, signer2, tf_erc20_addr):
     receiver_balance_bf = receiver_balance
 
 
-def edit_app_cfg(cli, i):
+def edit_app_cfg(cli, i, app_config={}):
```

Ruff (0.14.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)